src/Microsoft.PowerShell.Commands.Management/commands/management/Clipboard.cs

```diff
@@ -19,7 +19,8 @@ internal static class Clipboard
         private static string StartProcess(
             string tool,
             string args,
-            string stdin = "")
+            string stdin = "",
```

Comment:
```suggestion
            string stdin = string.Empty,
```

Reply:
Had to change this back, it needs to be a compile time constant.